src/Umbraco.Web/Security/MembershipHelper.cs

```diff
@@ -289,11 +289,46 @@ public virtual IPublishedContent GetByProviderKey(object key)
             return MemberCache.GetByProviderKey(key);
         }
 
+        public virtual IEnumerable<IPublishedContent> GetByProviderKey(IEnumerable<object> keys)
+        {
+            return keys?.Select(GetByProviderKey).WhereNotNull() ?? new IPublishedContent[0];
+        }
+
+        public virtual IEnumerable<IPublishedContent> GetByProviderKey(params object[] keys)
```

Comment:
I think with `params object[] keys` parameter, this will hide the other declared method `public virtual IPublishedContent GetByProviderKey(object key)`, perhaps we should make these methods distinguished by name, so instead call it `GetByProviderKeys` and pass in `IEnumerable<object>` ?

Reply:
Yup - see comment in the conversation below.

Reply:
Fixed in 64bae43